x/incentives/keeper/gauge.go

```diff
@@ -302,14 +307,20 @@ func (k Keeper) CreateGroup(ctx sdk.Context, coins sdk.Coins, numEpochPaidOver u
 func (k Keeper) chargeGroupCreationFeeIfNotWhitelisted(ctx sdk.Context, sender sdk.AccAddress) (chargedFee bool, err error) {
 	params := k.GetParams(ctx)
 	incentivesModuleAddress := k.ak.GetModuleAddress(types.ModuleName)
+
+	// don't charge fee if sender is the incentives module account
+	if sender.Equals(incentivesModuleAddress) {
+		return false, nil
+	}
```

Comment:
Please update unit test to make sure that the old bug is reproduced and caught by that test

Reply:
Added test case and confirmed it fails when fix is reverted https://github.com/osmosis-labs/osmosis/pull/6547/commits/96c6e31c92b45b5ce85d300769d4a5c316231d87